app/assets/stylesheets/home.scss

```diff
@@ -85,32 +73,32 @@
   font-family: Audiowide;
 }
 
-.rod-photo {
+.about-team-pic {
   width: 200px;
   height: 200px;
   margin: auto;
 }
 
-#rod1 {
-  background-image: url('rod1-cropped');
+#pic1 {
+  background-image: url('aboutteam1.jpg');
   background-size: cover;
   border-radius: 100px;
 }
 
-#rod2 {
-  background-image: url('rod2-cropped');
+#pic2 {
+  background-image: url('aboutteam2.jpg');
```

Comment:
we probably need to use `asset_path` for this to work in production. This may apply for the rest of them.